### ISSUE TITLE
Update to V1.01PRO

### ISSUE THREAD

**Issue by mbhassan** [krushkamx/nativescript-hprt#13]
I have the 1.0.1.1801PRO.jar for hprt. The jar is missing the header files though that this version has. Been trying to work on an updated version for this plugin but not having much luck. How did you generate the header files from the given jar?

**Comment by mbhassan** [comment]
Got the headers working. Is there any hope of any updates on this repo?

**Comment by davorpeic** [comment]
Hey,

yeah, update is coming soon, will work with latest 5.3 and with webpack. not sure will update the main library, need to see if there are any updates.

**Comment by mbhassan** [comment]
Attached the latest library I managed to get from the vendor.

[hprt.jar.zip](https://github.com/krushkamx/nativescript-hprt/files/3042627/hprt.jar.zip)

**Comment by davorpeic** [comment]
Ok, just to be sure, this is not a commercial version you buy or similar right, it is free to use?

**Comment by mbhassan** [comment]
yep its the free version. also available on http://www.hprt.com/Download.html . It didnt have the header files inside it so had to copy yours and create a new jar.
